src/BankConsole/Program.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using System;
 using System.Collections.Generic;
-using BankingSystem.BusinessLogic;
+using BankingSystem.Bank;
 
 namespace BankingSystem
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Dictionary<int, BankAccount> accounts = new Dictionary<int, BankAccount>();
-            Bank bank = new Bank(accounts);
-            BankService bankService = new BankService(bank);
+            CompareBankAccounts.Run();
+            
+            BankService bankService = new BankService(new Bank.Bank());
```

Comment:
Dont like the Bank.Bank() fix it :)

Reply:
I'd have problems to decide names, I will fix it